db/migrate/20170412141815_create_language_skills.rb

```diff
@@ -1,7 +1,7 @@
 class CreateLanguageSkills < ActiveRecord::Migration[5.1]
   def change
     create_table :language_skills do |t|
-      t.references :client, foreign_key: true
+      t.references :client
```

Comment:
The problem is, this is an old migration. Old migrations should never be touched. You need to change this with a migration, look into http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-remove_foreign_key for how to make this.